### PR TITLE
Adding account enrichment to malware

### PR DESCRIPTION
**Pull request by ssokolovich**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-hq.paloaltonetworks.local/browse/CIAC-3716

## Description
Adding Account Enrichment flow through the Account Enrichment Playbook.

## Screenshots
Malware_Investigation_ _Response_layout

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [x] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/CortexXDR/ReleaseNotes/4_8_16.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+
+#### Playbooks
+##### Cortex XDR Malware - Incident Enrichment
+Added Account Enrichment playbook.
```
Comment (altmannyarden) [inline Packs/CortexXDR/ReleaseNotes/4_8_16.md:4]: The playbook now supports account enrichment.

File: Packs/MalwareInvestigationAndResponse/README.md [hunk 2]
```diff
@@ -22,7 +22,7 @@ The pack closely maps evidence to MITRE ATT&CK to uncover evidence of:
 * Supports aggregating incidents from EDRs, including multiple alerts and files.
 * Supports fetching malware incidents either from the EDR product or from a SIEM solution.
```
Comment (altmannyarden) [inline Packs/MalwareInvestigationAndResponse/README.md:24]: lets add a line here for the account enrichment as well

File: Packs/CortexXDR/ReleaseNotes/4_8_16.md (hunk 1 above)
Comment (ssokolovich) [reply Packs/CortexXDR/ReleaseNotes/4_8_16.md:4]: Done

File: Packs/MalwareInvestigationAndResponse/README.md (hunk 2 above)
Comment (ssokolovich) [reply Packs/MalwareInvestigationAndResponse/README.md:24]: Done

**Review by altmannyarden — APPROVED** [review]

**Pull request merged**